src/environment.ts

```diff
@@ -10,15 +10,21 @@ export enum PlatformEnvironment {
 
 export const DEFAULT_ENVIRONMENT = PlatformEnvironment.Prod as const;
 export const DEFAULT_REGION = Region.US as const;
+export const DEFAULT_RETRY_AFTER = 5000;
+export const DEFAULT_EJECT_AFTER = 3600000;
 
 export type PlatformUrlOptions = {
   environment?: PlatformEnvironment;
   region?: Region;
+  retryAfter?: number;
+  ejectAfter?: number;
 };
 
 const defaultOptions: PlatformUrlOptions = {
   environment: DEFAULT_ENVIRONMENT,
   region: DEFAULT_REGION,
+  retryAfter: DEFAULT_RETRY_AFTER,
```

Comment:
Why not export the entire `defaultOptions` and reuse it as a whole instead of exporting and importing parts of it?